packages/solo/test/test-home.js

```diff
@@ -1,16 +1,19 @@
 /* global process */
 
-import '@endo/init/pre-bundle-source.js';
-
 // `test.after.always` does not yet seem compatible with ses-ava
 // See https://github.com/endojs/endo/issues/647
 // TODO restore
-// import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava';
```

Comment:
Drive-by fix for the `.js` extension, left over before the RESM-to-NESM migration.